### PR TITLE
Script to auto-detect WiFi adapters

### PR DESCRIPTION
**Pull request by ramkrishna757575**

### CODE REVIEW

File: wifi-adapters/init-nics.service [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Rename Wireless Interface
+After=network.target
+
+[Service]
+ExecStart=/home/radxa/scripts/autoload-wfb-nics.sh
```
[the remark this answers is not in the document]
Comment (ramkrishna757575) [reply wifi-adapters/init-nics.service:6]: Makes sense. I'll make these changes

[the remark this answers is not in the document]
Comment (mikecarr) [reply wifi-adapters/init-nics.service:6]: Is there a repo for the Radxa base image?

[the remark this answers is not in the document]
Comment (ramkrishna757575) [reply wifi-adapters/init-nics.service:6]: Not sure about the base image but I found this https://radxaos-sdk.github.io/rsdk/

[the remark this answers is not in the document]
Comment (JohnDGodwin) [reply wifi-adapters/init-nics.service:6]: > Is there a repo for the Radxa base image?

Radxa's official images are [here](https://github.com/radxa-build/radxa-zero3/releases).
Only the debian xfce image is officially working. 
If you need a base cli image, radxa has a tool to generate one called rbuild. [Read about it here.](https://radxa-repo.github.io/rbuild/)

**Comment by luastoned** [comment]
@ramkrishna757575 i think your merge requests should be made towards https://github.com/OpenIPC/sbc-groundstations
I think since 1.7 this has been integrated as well?

**Comment by seriyps** [comment]
@luastoned yes, it is for sure included in 1.7.0

**Pull request closed without merging**